README.md

```diff
@@ -52,13 +52,13 @@ Then you should access your Auth0 tenant to configure 1 manual steps:
 
 ### Required: Running the products A, B, C, and the API
 
-Please configure all the variables in the section `Auth0 Management API stuff` in the [.env](./.env) file. To configure all `env.development` files, execute the command:
+Please configure all the variables in the section `Auth0 Management API stuff` in the [.env](./.env) file. It is also necessary to change the `YOUR_TENANT` string in other files like [file1](https://github.com/juntossomosmais/auth0-samples/blob/main/orchestrator/platform/grants/Management%20-%20Orchestrate%20- %20Django%20API%20(https---antunes.us.auth0.com-api-v2-).json) and [file2](https://github.com/juntossomosmais/auth0-samples/blob/main/ orchestrator/platform/grants/Product%20A%20(https---antunes.us.auth0.com-api-v2-).json) to work correctly. To configure all `env.development` files, execute the command:
```

Comment:
**issue**: The markdown does not render properly.

Reply:
@willianantunes fixed:
![image](https://user-images.githubusercontent.com/853089/236007180-634fd0a9-f907-4b78-822b-b6ee71c8040a.png)